importer/tests/test_xanalytics.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for hitting the xanalytics Celery tasks.
+"""
+
+import json
+import logging
+
+from django.conf import settings
+import responses
+
+from importer.tasks import populate_xanalytics_fields, check_for_results
+from learningresources.tests.base import LoreTestCase
+
+log = logging.getLogger(__name__)
+# pylint: disable=no-self-use
+
+
+class TestAPI(LoreTestCase):
```

Comment:
There aren't any unit tests for xanalytics/send_request, get_result, get_xanalytics_file, or learningresources/api.py:update_xanalytics, just integration level tests for check_for_results and populate_xanalytics_fields, they should be added